profile/README.md

```diff
@@ -21,16 +21,19 @@ Camunda Platform 8 is comprised of the following components:
 * Web Modeler - Collaborate and model processes, deploy and start new instances all without leaving Camunda Platform 8.
```

Comment:
Is there a reason why we only link to some of these? Should we link to the documentation for the others?

Reply:
Public vs. Private/Internal repos. The links are only to publicly available repos.